ivn/core.py

```diff
@@ -512,19 +515,23 @@ def create_interface(self):
                 self.logger_topo.warning("ip link {} exists in namespace {} so not create it.".
                                          format(ifname, self.__namespace))
                 return
+        IP_ROUTE.close()
```

Comment:
better to move this line just before line 511.  "return" in 517 may jump out before calling IP_ROUTE.close()